src/main/kotlin/uk/gov/dluhc/printapi/service/SftpService.kt

```diff
@@ -0,0 +1,19 @@
+package uk.gov.dluhc.printapi.service
+
+import org.springframework.integration.file.FileHeaders
+import org.springframework.integration.sftp.session.SftpRemoteFileTemplate
+import org.springframework.integration.support.MessageBuilder
+import org.springframework.stereotype.Service
+import java.io.InputStream
+
+@Service
+class SftpService(val sftpTemplate: SftpRemoteFileTemplate) {
+
+    fun sendFile(inputStream: InputStream, filename: String): String =
+        sftpTemplate.send(
+            MessageBuilder
+                .withPayload(inputStream)
+                .setHeader(FileHeaders.FILENAME, filename)
+                .build()
+        )
+}
```

Comment:
I would expect `ktlint` to fail the build as there is no new line at end of this class 🤔

Reply:
New line does exist 🤷